app/assets/stylesheets/custom.css.scss

```diff
@@ -1,3 +1,15 @@
+.logo {
+  text-transform: uppercase;
+  font-weight: 700;
+  font-size: 16px;
+  color: black;
+  font-family: Open Sans, Arial;
+  &:hover {
+    color: black;

```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.